packages/oauth/src/createPublicOauthClient.ts

```diff
@@ -151,6 +151,10 @@ export function createPublicOauthClient(
   const authServer = createAuthorizationServer(ctxPath, url);
   const oauthHttpOptions: HttpRequestOptions = { [customFetch]: fetchFn };
 
+  if (scopes.length === 0) {
```

Comment:
guessing this actually belongs in the `processOptionsAndAssignDefaults`?

Reply:
Oops, yeah, thanks!